src/Import/src/FMI3/fmi3_import.c

```diff
@@ -407,11 +407,21 @@ fmi3_import_variable_list_t* fmi3_import_get_continuous_state_derivatives_list(f
     return fmi3_import_vector_to_varlist(fmu, fmi3_xml_get_continuous_state_derivatives(fmi3_xml_get_model_structure(fmu->md)));
 }
 
+fmi3_import_variable_list_t* fmi3_import_get_clocked_states_list(fmi3_import_t* fmu){
+    if(!fmi3_import_check_has_FMU(fmu)) return 0;
```

Comment:
```suggestion
    if (!fmi3_import_check_has_FMU(fmu)) return NULL;
```
If you touch code that has `if(` or `for(` without space, feel to fix it and any code close by that does it. And the same for return 0; -> return NULL;.